src/pybind/mgr/dashboard/frontend/src/app/shared/api/rbd.service.ts

```diff
@@ -0,0 +1,110 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class RbdService {
+
+  constructor(private http: HttpClient) {
```

Comment:
Consider returning Observables here like [this service](https://github.com/ceph/ceph/pull/21351/files#diff-0a729d63f185cb3c722b720d9738bca2R10).

Reply:
This service is now returning Observables.